README.md

```diff
@@ -105,6 +105,8 @@ Flags:
                        - an ID of a previous measurement to run with its probes
                        (default "world")
   -h, --help          help for globalping
+  -4, --ipv4          resolve names to IPv4 addresses
```

Comment:
@MartinKolarik v4 remains the default right? We should mention it here that if not specified we use v4

Reply:
Right now yes but I wouldn't want people to rely on it. If we add auto selection later then that would be the default.

Actually now that I think about it, we might have overestimated how hard that would be. Let me come back to that in https://github.com/jsdelivr/globalping/issues/529.

Reply:
Ok but until we add auto, we should default to v4 and make sure it's optional to specify the version for better ux

Reply:
It is optional to specify it of course, I just wouldn't explicitly document it.

Reply:
if `--ipv4` is the default option, do we need to have a flag for it? or am I missing something?

Reply:
With the flag the behavior is guaranteed, the default may change.